src/chains/filecoin/filecoin/src/blockchain.ts

```diff
@@ -80,6 +80,10 @@ export default class Blockchain extends Emittery.Typed<
 
   readonly #database: Database;
 
+  get dbDirectory(): string | null {
+    return this.#database.directory;
+  }
+
```

Comment:
Do we need this anywhere?

Reply:
It's needed in [Ganache UI](https://github.com/trufflesuite/ganache/blob/feat/filecoin-misc/static/node/chain/chain-v7.js#L133) which I believe is needed to support workspaces; I can add a comment